### PR TITLE
Use Qt shadow build, and try to warn about WebEngine config failure

### PR DESCRIPTION
**Pull request by ihnorton**
Fixes #36, #40

### CODE REVIEW

**Comment by ihnorton** [comment]
Heh, the webengine check [does work](https://circleci.com/gh/jcfr/qt-easy-build/70?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) (CI environment is missing some dependencies). I pushed another commit to make the check non-fatal, because it doesn't really need to be. The idea is just to let people know that webengine wasn't built, why, and hopefully how to get it to build without scrolling back thousands of lines.

**Comment by ihnorton** [comment]
The shadow part of this PR probably has to be tabled for now. There is an issue with private includes in a shadow build:

  https://bugreports.qt.io/browse/QTBUG-37417

which leads to

> /mnt/bld/s5db/sb/CTK-build/PythonQt/src/PythonQt.cpp:1951:42: fatal error: private/qmetaobjectbuilder_p.h: No such file or directory

unless we want to carry some form of the patches linked in that issue.

File: Build-qt.sh [hunk 1]
```diff
@@ -147,13 +151,16 @@ cwd=$(pwd)
...
 src_dir="$cwd/qt-everywhere-src-$QT_VERSION"
 
+# Build directory
+bld_dir="$cwd/qt-everywhere-build-$QT_VERSION"
+
 # Install directory
 if [[ -z $install_dir ]]
 then
-  install_dir="$cwd/qt-everywhere-build-$QT_VERSION"
+  install_dir="$cwd/qt-everywhere-$QT_VERSION"
```
Comment (jcfr) [inline Build-qt.sh:163]: Should we have the `install` word in the name of the directory ?

For example `install_dir="$cwd/qt-everywhere-install-$QT_VERSION"`

**Comment by jcfr** [comment]
What do you mean by "shadow build" ? I didn't see a second build tree.

Also, in a follow up set of changes, would it help to move the `5.10` branch as unmaintained and focus on `5.11` ?

**Comment by jcfr** [comment]
@jamesobutler  I would be curious to get your feedback on  this one.

**Comment by jamesobutler** [comment]
hmm I am familar with that QtWebEngine can silently fail to build.  This was seemingly addressed in https://github.com/jcfr/qt-easy-build/commit/756c6e5a830585e3bc0214c3d99b87404fb55e7c

**Comment by jcfr** [comment]
Thanks for the feedback, I closing this for now.

**Pull request closed without merging**